### ISSUE TITLE
Width capped at 90% and height capped at 160px in javascript/building-blocks/tasks/styles.css

### ISSUE THREAD

**Issue by bsmth** [mdn/learning-area#681]
Why do we have [width capped at 90%](https://github.com/mdn/learning-area/blob/c754e63586d8d897ebee120fe4dcd320e1eefef1/javascript/building-blocks/tasks/styles.css#L41) and [height capped at 160px](https://github.com/mdn/learning-area/blob/c754e63586d8d897ebee120fe4dcd320e1eefef1/javascript/oojs/tasks/styles.css#L52)?
> ![_1](https://github.com/mdn/content/assets/87750369/477a57f0-767e-407a-9e59-9e18d96b6356)

_Originally posted by @OnkarRuikar in https://github.com/mdn/content/issues/29670#issuecomment-1765527133_

**Comment by bsmth** [comment]
@OnkarRuikar I was also curious about this one. Creating a tracking issue here for it 👍🏻

**Comment by OnkarRuikar** [comment]
> @OnkarRuikar I was also curious about this one. Creating a tracking issue here for it 👍🏻

This can easily be fixed but we need to know why it is this way to avoid regression.

**Comment by bsmth** [comment]
> we need to know why it is this way

I have a strong suspicion this is a leftover from previous site layouts. I've asked the team and there are no leads at the moment as to the rationale, it would be good to check with eng about it.

**Comment by bsmth** [comment]
I don't think this applies anymore. You can check https://developer.mozilla.org/en-US/docs/Learn_web_development/Core/Scripting/Test_your_skills/Conditionals
